README.md

```diff
@@ -110,6 +110,19 @@ HOSTNAME_COMMAND=wget -t3 -T2 -qO-  http://169.254.169.254/latest/meta-data/loca
 ```
 Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
 
+For Local, OpenShift, Kubernetes or Swarm deployments you can use the following to get the container host's IP:
+```
+HOSTNAME_COMMAND=ip -o -4 addr list eth0 | awk '{print \$4}' | cut -d/ -f1
```

Comment:
This doesn't look like it provides the 'container host's IP' to me. As far as I can tell this returns the internal IP assigned to the container within the assigned docker network (not normally route-able directly from the host)

Reply:
The internal IP is available/routable to other containers within the same network. This can be useful as the kafka advertized IP/hostname that is provided to zookeeper and kafka clients that obtain the kafka hosts/IPs dynamically. That aside, I've found this useful for testing, where docker-compose is used to start zookeeper and kafka, and the test is started in it's own process.

Reply:
1) The comment is ambiguous. I interpret "container host's IP" as the IP-of-the-host where the container is running. I think you just mean container IP?
2) From your comment, this particular configuration is useful to your use-case but it's not a universal approach - or one suitable if for example you wanted to have access outside of the docker-network.
3) I think by default, the image will do what you want and advertise the docker assigned hostname. Specifying `listeners` of `PLAINTEXT://:9092` should also give you what you want I believe.